docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,290 @@
+apiVersion: k8s.nginx.org/v1    
+kind: VirtualServer    
+metadata:    
+  name: cafe    
+  namespace: nginx-ingress
+spec:    
+  host: cafe.example.com    
+  tls:    
+    secret: cafe-secret    
+  upstreams:    
+  - name: tea    
+    service: tea-svc    
+    port: 80    
+    use-cluster-ip: true
+  - name: coffee    
+    service: coffee-svc    
+    port: 80    
+    use-cluster-ip: true
+  routes:    
+  - path: /tea    
+    action:
+      proxy:
+        upstream: tea
+        requestHeaders:
+          set:
+          - name: Host
+            value: tea-svc.nginx-ingress.svc.cluster.local
+  - path: /coffee
+    action:
+      proxy:
+        upstream: coffee
+        requestHeaders:
+          set:
+          - name: Host
+            value: coffee-svc.nginx-ingress.svc.cluster.local
+```
+
+With our new Host header control in v1.11, when VirtualServer is configured with `requestHeaders`, the value specified will be used and `proxy_set_header $host` will NOT be used. By enabling `use-cluster-ip` to **true**, NGINX will forward requests to the service IP. In our above example, that would be `tea-svc` and `coffee-svc`.
+
+Here is the output of `nginx -T` to show our upstreams and proxy_set_header values 
+The server in the upstream is the IP address of the service for that given application)
+
+```bash
+upstream vs_nginx-ingress_cafe_tea {
+    zone vs_nginx-ingress_cafe_tea 256k;
+    random two least_conn;
+    server 10.96.222.104:80 max_fails=1 fail_timeout=10s max_conns=0;
+}
+
+upstream vs_nginx-ingress_cafe_coffee {
+    zone vs_nginx-ingress_cafe_coffee 256k;
+    random two least_conn;
+    server 10.96.252.249:80 max_fails=1 fail_timeout=10s max_conns=0;
+}
+
+server {
+    listen 80;
+        
+    location /tea {
+
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $vs_connection_header;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Port $server_port;
+        proxy_set_header X-Forwarded-Proto $scheme;
+            
+        proxy_set_header Host "tea-svc.nginx-ingress.svc.cluster.local";
+    }
+    
+    location /coffee {
+ 
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $vs_connection_header;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Port $server_port;
+        proxy_set_header X-Forwarded-Proto $scheme;
+            
+        proxy_set_header Host "coffee-svc.nginx-ingress.svc.cluster.local";
+    }   
+}
+```
+
+Notice the abscence of `proxy_set_header $host`. This is because we are using `action-proxy-requestHeaders` option to specify what headers we want to pass to Istio sidecar. This is required by Istio.

```

Comment:
Why do we need to describe this?
If there are settings specific to success with Istio, then we should briefly mention that at the beginning.

This might be useful detail for a troubleshooting section if it relates to things not functioning or functioning abnormally.

Reply:
We do not. It is more of what happens under the hood. Troubelshooting might make more sense.